CONTRIBUTING.md

```diff
@@ -65,6 +65,20 @@ the API interactions should be well understood and there should be no
 technical surprises; we expect the only discussion necessary on PRs to be
 for wordsmithing and formatting.
 
+## Github Labels
+
+There is a label for each phase of the SDLC:
+
+- `reviewing proposal`: The API working group is reviewing a proposed change
+to the API.
+- `proposal finalized`: (??)
+- `validating through implementation`: One or more platforms that integrate
+   with the API implement the proposed change to provide feedback on the design
+- `validated by implementation`: (??)
```

Comment:
Not all features will become Alpha first. 

Once the team that has implemented the feature believes the work proves the design is viable, should they open a PR right away? I imagine we might want a discussion about it, but that isn't to say a PR couldn't be opened. I'd be ok going directly from `validating through implementation` to `pr submitted`.

We should also consider the possibility that the design needs reworking, and so the state may go back to `reviewing proposal`